### ISSUE TITLE
Missing with torch.no_grad() Block in Testing Phase

### ISSUE THREAD

**Issue by 943fansi** [xiyuanzh/PILOT#1]
In the testing phase, is it missing the `with torch.no_grad():` block?

<pre>
<code>
# test
total_loss, total_vi_loss, total_w_loss, total_a_loss = 0, 0, 0, 0
denoise_vi_list, vi_w_list, vi_a_list = [], [], []
<b><font color="red">with torch.no_grad():</font></b>
    for batch_vi, batch_w, batch_a in DataBatch(low_vi, low_w, low_a, args.batchSize, shuffle=False):
</code>
</pre>

**Comment by xiyuanzh** [comment]
Thanks for the comment! I just updated the code.
